lib/DDG/Spice/DebVersion.pm

```diff
@@ -1,9 +1,18 @@
-# Get the version of a package in various debian repositories
 package DDG::Spice::DebVersion;
 
```

Comment:
It wouldn't hurt to set an ABSTRACT, but we actually use the `description` (or will be soon) when the IA Pages (duck.co/ia) are further along.